webapp/sources/rudder/rudder-web/src/main/elm/editor/sources/JsonDecoder.elm

```diff
@@ -82,7 +82,7 @@ decodeMethodCall =
     |> required "parameters"  (list decodeCallParameter )
     |> required "class_context"  (map parseCondition string)
     |> required "component"  string
-    |> required "disableReporting" bool
+    |> optional "disableReporting" bool False
```

Comment:
that was the main problem: a new field was added, but old techniques don't have it.